docs/examples.md

```diff
@@ -420,6 +420,27 @@ object PerftConverter extends IOApp.Simple {
 ```
 @:@
 
+## A helper utility to write out an `Seq[CaseClass]` to CSV
+
+In the presence of a CsvRowEncoder
+
+```
```

Comment:
So that the code gets compiled in CI. Also see the snippets above, which have versions for Scala 2 and Scala 3.

```suggestion
```mdoc:reset:silent
```